### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by AleksandrovaP**

### CODE REVIEW

File: src/script/Bullet.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   private posTopLeftY:number;
+   private parent:Element;
+
+   constructor(width:number, height:number, posTopLeftX:number, posTopLeftY:number) {
+      this.width = width;
+      this.height = height;
+      this.posTopLeftX = posTopLeftX;
+      this.posTopLeftY = posTopLeftY;
+
+      this.domElem = document.createElement("img");
```
Comment (alex-jordanova) [inline src/script/Bullet.ts]: Isn't the 'this' keyword unnecessary when you don't have to distinguish the class variable from a passed down parameter?

File: src/script/BulletManager.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+   private generateBullet():Bullet {
+      let length = this.bullets.length;
+      this.bullets[length] = new Bullet(WIDTH, HEIGHT, POS_TOP_LEFT_X, POS_TOP_LEFT_Y);
+      return this.bullets[length];
+   }
+
+   private startSingleBullet(game:Game):void {
+      let bullet = this.generateBullet();
+      let singleInterval = setInterval(() => {
+         bullet.move(50);
```
Comment (alex-jordanova) [inline src/script/BulletManager.ts]: 50 what?

File: src/script/BulletManager.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+   stopBullets():void {
+      clearInterval(this.generateInterval);
+      let size = this.bullets.length;
+      for (let i = 0; i < size; ++i) {
+         this.deleteBullet(this.bullets[0]);
+      }
+   }
+
+   private reachedEnd(bullet:Bullet):boolean {
+      return bullet.getPosTopLeftX() <= -50;
```
Comment (alex-jordanova) [inline src/script/BulletManager.ts]: Magical number.

File: src/script/Bullet.ts (hunk 1 above)
Comment (AleksandrovaP) [reply src/script/Bullet.ts]: In JS and respectively in TS 'this' has different meaning

**Pull request merged**